httpie/input.py

```diff
@@ -328,6 +328,22 @@ def _parse_items(self):
             self.args.files = items.files
             self.args.params = items.params
 
+        if self.args.files and '-' in self.args.files.values():
```

Comment:
Does `cURL` allow for you to do something like:

```
date | curl ... file@- file@- file@-
```

If so, does this allow HTTPie to work the same way?